__tests__/reporters/_mock.js

```diff
@@ -16,7 +16,8 @@ type MockCallback = (reporter: Reporter, opts: Object) => ?Promise<void>;
 export default function<T>(
   Reporter: Function,
   interceptor: Interceptor<T>,
-  prepare?: (reporter: Reporter) => any,
+  prepare?: ?(reporter: Reporter) => any,
+  opts?: Object,
```

Comment:
nit: this can be `extraOpts` and then you can avoid the `opts` -> `newOpts` rename.

Reply:
Yeah, I was thinking that too. But thought it would have been preferred to keep the argument named as `opts`. Let me know if you want to change though 😄

Reply:
Nope, already merged :D